includes/admin/reporting/class-sales-logs-list-table.php

```diff
@@ -69,7 +69,7 @@ public function column_default( $item, $column_name ) {
 
 			case 'user_id' :
 				return '<a href="' .
-					admin_url( 'edit.php?post_type=download&page=edd-payment-history&user=' . urlencode( $item['user_id'] ) ) .
+				       admin_url( 'edit.php?post_type=download&page=edd-payment-history&user=' . ( ! empty( $item['user_id'] ) ? urlencode( $item['user_id'] ) : edd_get_payment_user_email( $item['payment_id'] ) ) ) .
```

Comment:
No need for an if/else, just setup the `user` parameter with a ternary (like you have) on the line above.